### PR TITLE
Cleans up late joining, fixes admins failing to join.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Fixes a bug that prevents admins from joining while the popcap is restricting people.
Cleans up some of the code.

## Why It's Good For The Game

Fixes a bug and makes the checks look a bit nicer.

## Changelog
:cl:
fix: Fixes admins being unable to bypass popcap.
/:cl:

### CODE REVIEW

File: code/modules/mob/dead/new_player/new_player.dm [hunk 1]
```diff
@@ -140,10 +140,13 @@
...
 			return
 
-		if(SSticker.queued_players.len || (relevant_cap && living_player_count() >= relevant_cap && !(ckey(key) in GLOB.admin_datums)))
+		if(SSticker.queued_players.len || (relevant_cap && living_player_count() >= relevant_cap))
 			if(IS_PATRON(src.ckey))
 				LateChoices()
 				return
+			if(client in GLOB.admins)
+				LateChoices()
+				return
```
Comment (park66665) [inline code/modules/mob/dead/new_player/new_player.dm]:
```suggestion
			if(IS_PATRON(src.ckey) || client in GLOB.admins)
				LateChoices()
				return
```

~~Fastest review in Bee~~

Comment (PowerfulBacon) [reply code/modules/mob/dead/new_player/new_player.dm]: ew I dont really like combining but I guess

Comment (PowerfulBacon) [reply code/modules/mob/dead/new_player/new_player.dm]:
```suggestion
			if(IS_PATRON(src.ckey) || (client in GLOB.admins))
				LateChoices()
				return
```

**Review by park66665 — APPROVED** [review]
All clear. This PR is good to go.

**Pull request merged**